docs/1-introduction/1-1-getting-started/1-1-1-hello-window.md

```diff
@@ -402,22 +402,22 @@ void Application::Run()
 }
 ```
 
-You can clearly see what it is doing. `Initialize`, as the name suggests, will initialize everything
-which is required for the app to run, which currently is the window in our case. In future chapters it
-will also including initializing `D3D11` and its resources or imgui for some UI.
+This function is pretty simple. `Initialize`, as the name suggests, will initialize everything
+which is required for the app to run, which currently is the window in our case. In future
+chapters it will also include initializing `D3D11`, its resources, and ImGUI for the UI.
 
-`Load`s purpose is to load all the assets required to run the application, in further chapters it will
+`Load`'s purpose is to load all the assets required to run the application, in further chapters it will
 encompass textures, shaders, models and other things.
 
-The next block is the aforementioned mainloop or gameloop, which still does what it was doing before,
-checking with the OS if events need to be processed and now we also call a `Update` and `Render` method.
+The next block is the aforementioned mainloop or game loop, which still does what it was doing before,
+checking with the OS if events need to be processed, and now we also call a `Update` and `Render` method.
```

Comment:
hmm same here: comma before and. That should be illegal even in English language (although it very likely is legal English)